examples/TreeAnalysis.jl

```diff
@@ -75,4 +95,24 @@ ccolamd_tree = buildTreeFromOrdering!(deepcopy(fg), ccolamd_ordering)
 ccolamd_tree_nnz = nnzTree(ccolamd_tree)
 ccolamd_tree_cost02 = getTreeCost_02(ccolamd_tree)
 
-# Plot both data points.
+# Plot data points and underlying histogram.
+bincnt = 20
+layers = []
+push!(layers, Gadfly.layer(x=[amd_tree_nnz],
+                           y=[amd_tree_cost02],
+                           Theme(default_color=colorant"green")))
```

Comment:
Just a note on Colors.jl -- it's a standardized package throughout the entire ecosystem.  In other words conversion between color types -- number represenration (UInt8,16,32, Float32,Float64, FixedPoint numbers etc etc etc, all standard).  This colorant"green" approach is just a handy shorthand.  For more details, this might be a good place to drop in:
http://juliagraphics.github.io/Colors.jl/stable/namedcolors/

cc @tonioteran